[2023-03-25T03:42:33.053+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v02.get_sklearn scheduled__2023-03-24T00:00:00+00:00 [queued]>
[2023-03-25T03:42:33.084+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v02.get_sklearn scheduled__2023-03-24T00:00:00+00:00 [queued]>
[2023-03-25T03:42:33.085+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-25T03:42:33.095+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-03-25T03:42:33.096+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-25T03:42:33.175+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): get_sklearn> on 2023-03-24 00:00:00+00:00
[2023-03-25T03:42:33.182+0000] {standard_task_runner.py:55} INFO - Started process 189 to run task
[2023-03-25T03:42:33.193+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_dependencies_v02', 'get_sklearn', 'scheduled__2023-03-24T00:00:00+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_dependencies.py', '--cfg-path', '/tmp/tmp2d3rq0v4']
[2023-03-25T03:42:33.202+0000] {standard_task_runner.py:83} INFO - Job 247: Subtask get_sklearn
[2023-03-25T03:42:33.755+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_python_dependencies_v02.get_sklearn scheduled__2023-03-24T00:00:00+00:00 [running]> on host 9783a48d7fca
[2023-03-25T03:42:34.324+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ThinhHuynh
AIRFLOW_CTX_DAG_ID=dag_with_python_dependencies_v02
AIRFLOW_CTX_TASK_ID=get_sklearn
AIRFLOW_CTX_EXECUTION_DATE=2023-03-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-24T00:00:00+00:00
[2023-03-25T03:42:34.338+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_python_dependencies.py", line 14, in get_sklearn
    import sklearn
ModuleNotFoundError: No module named 'sklearn'
[2023-03-25T03:42:34.415+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=dag_with_python_dependencies_v02, task_id=get_sklearn, execution_date=20230324T000000, start_date=20230325T034233, end_date=20230325T034234
[2023-03-25T03:42:34.529+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 247 for task get_sklearn (No module named 'sklearn'; 189)
[2023-03-25T03:42:34.572+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-25T03:42:34.783+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
